frames/src/main/java/com/checkout/frames/component/cardholdername/CardHolderNameViewModel.kt

```diff
@@ -0,0 +1,66 @@
+package com.checkout.frames.component.cardholdername
+
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.ui.text.input.ImeAction
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.checkout.base.mapper.Mapper
+import com.checkout.frames.component.base.InputComponentState
+import com.checkout.frames.di.base.InjectionClient
+import com.checkout.frames.di.base.Injector
+import com.checkout.frames.di.component.CardHolderNameViewModelSubComponent
+import com.checkout.frames.style.component.CardHolderNameComponentStyle
+import com.checkout.frames.style.component.base.InputComponentStyle
+import com.checkout.frames.style.view.InputComponentViewStyle
+import javax.inject.Inject
+import javax.inject.Provider
+
+internal class CardHolderNameViewModel @Inject constructor(
```

Comment:
Shall we `cardholder` or `card holder`? There's already some text using `cardholder` as a single word in the codebase. Maybe we should align them all using either one of them?

Reply:
All the places have cardholder names as a description for the test case names. Happy to change there